src/master.rs

```diff
@@ -336,6 +373,15 @@ impl Master {
     // reference_clock_time, sync_monitor_queue, sync_monitor_process
 }
 
+fn raw_array_to_string(arr: &[i8; 64]) -> String {
+    let s = arr
+        .iter()
+        .filter(|x| **x != 0)
+        .map(|x| *x as u8)
+        .collect::<Vec<_>>();
+    String::from_utf8_lossy(&s).to_string()
```

Comment:
I'm not sure of the encoding here, although it's probably not UTF-8 :)  I suspect latin1 or windows1252.

Reply:
good point. thx.